impl/maven-executor/src/main/java/org/apache/maven/cling/executor/internal/ToolboxTool.java

```diff
@@ -142,7 +142,7 @@ private String validateOutput(boolean shave, ByteArrayOutputStream stdout, ByteA
         }
         // sanity checks: stderr has any OR result is empty string (no method should emit empty string)
         if (stderr.size() > 0 || result.trim().isEmpty()) {
-            throw new ExecutorException(
+            System.err.println(
```

Comment:
Why?

Reply:
To not have it lost. Without this, the output is lost, this way you have it at least in surefire report.
And with throw it would just fail (as on Java 24 mvn4 always emits warning on stderr).

Reply:
I see, makes sense.

Reply:
leftover...

<img width="1090" alt="image" src="https://github.com/user-attachments/assets/2aeb21ec-12ea-4409-84fa-39d3cf65416a" />

thats why we to rely on tools, not on human try&error:
https://docs.openrewrite.org/recipes/staticanalysis/unnecessarythrows